openassessment/data.py

```diff
@@ -774,11 +774,17 @@ def collect_ora2_submission_files(cls, course_id):
         all_submission_information = sub_api.get_all_course_submission_information(course_id, 'openassessment')
 
         for student, submission, _ in all_submission_information:
-            answer = submission['answer']
+            answer = submission.get('answer', dict())
 
             # collecting submission attachments with metadata
             for index, file_key in enumerate(answer.get('file_keys', [])):
-                file_name = answer['files_names'][index]
+                file_names = answer.get('files_names', answer.get('files_name', []))
```

Comment:
I was confused by this on the first glance and then noticed files_name**s** vs files_name :(